Lidgren.Network/Platform/PlatformWin32.cs

```diff
@@ -71,7 +72,7 @@ private static IPAddress ProbeDefaultRouteAddress()
 				// This basically gets us the network interface address "that goes to the router".
 				using var socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp);
 				socket.Connect(new IPAddress(new byte[] { 1, 1, 1, 1 }), 12345);
-				return ((IPEndPoint)socket.LocalEndPoint).Address;
+				return (socket.LocalEndPoint as IPEndPoint)?.Address;
```

Comment:
`LocalEndPoint` is expected to be set by the connect call, this should be an assert if anything at all.